docs/documentation/upgrading/topics/changes/changes-26_2_0.adoc

```diff
@@ -0,0 +1,7 @@
+= Removal of XSS Header
+
+We have removed the X-XSS-Protection response header from Keycloak, this header was a feature of Internet Explorer,
+Chrome and Safari that stopped pages from loading when they detected reflected cross-site scripting (XSS) attacks.
+
+Since, this header is no longer supported in any of the user agents supported by Keycloak and has been replaced by
+Content-Security-Policy. More details about the header can be found https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection[here]
```

Comment:
```suggestion
Since this header is no longer supported in any of the user agents supported by Keycloak, it has been replaced by
Content-Security-Policy. For more details, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection[X-XSS-Protection].
```